docs/source/overview.rst

```diff
@@ -239,7 +239,8 @@ This functionality is optional and now part of the `ipyparallel
 Portability and Python requirements
 -----------------------------------
 
-As of the 2.0 release, IPython works with Python 2.7 and 3.3 or above.
+Version 6.0 of IPython work with python 3.3 and above.
```

Comment:
Version 6.0+ supports compatibility with Python 3.3 and higher.
Versions 2.0 to 5.x work with Python 2.7.x releases and Python 3.3 and higher.